xmodule/x_module.py

```diff
@@ -519,7 +519,7 @@ def get_child(self, usage_id):
         child.runtime.export_fs = self.runtime.export_fs
         return child
 
-    def get_required_module_descriptors(self):
+    def get_required_block_descriptors(self):
         """Returns a list of XModuleDescriptor instances upon which this module depends, but are
         not children of this module"""
```

Comment:
```suggestion
    def get_required_block_descriptors(self):
        """
        Return a list of XBlock instances upon which this block depends but are
        not children of this block.
        
        TODO: Move this method directly to the ConditionalBlock.
        """
```

This isn't used anywhere else, so let's add a note about it.